coreapi/sfdc/src/main/java/com/jalasoft/sfdc/ui/pages/priceBooksFormPage/PriceBooksFormPageClassic.java

```diff
@@ -0,0 +1,43 @@
+package com.jalasoft.sfdc.ui.pages.priceBooksFormPage;
+
+import com.jalasoft.sfdc.ui.pages.contactDetailPage.ContactDetailPageClassic;
+import com.jalasoft.sfdc.ui.pages.priceBooksDetailPage.PriceBooksDetailPage;
+import com.jalasoft.sfdc.ui.pages.priceBooksDetailPage.PriceBooksDetailPageClassic;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PriceBooksFormPageClassic extends PriceBooksFormPage {
+    @FindBy(id = "Name")
+    WebElement priceBooksName;
+
+    @FindBy(id = "Description")
+    WebElement description;
+
+    @FindBy(xpath = ".//*[@id='bottomButtonRow']/input[1]")
+    WebElement clickBtnSave;
+
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
+
+    }
+    public void setName(final String name) {
```

Comment:
you cant the change for this method:
@Override
    public PriceBooksDetailPage insertPriceBooks(String priceBookName, String description) {
        driverTools.setInputField(priceBooksName, priceBookName);
        driverTools.setInputField(description, description);
        return new PriceBooksDetailPageClassic();
    }